framework/src/modules/chain/components/storage/entities/account.js

```diff
@@ -163,7 +124,9 @@ class ChainAccount extends AccountEntity {
 		}
 
 		accounts = accounts.map(account => {
-			const parsedAccount = _.defaults(account, defaultCreateValues);
+			let parsedAccount = _.defaults(account, defaultCreateValues);
+			parsedAccount = ChainAccount._stringifyVotedDelegates(parsedAccount);
```

Comment:
Is this a static private method?
If only used in this file, maybe create as function instead of method?

Reply:
@shuse2 made the same comment but on this file we're already using static methods. For now let's keep like this for this to be merged please create an issue for refactoring whole file maybe if you think it should be functions?